submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -11,13 +11,16 @@ import ebi.ac.uk.model.ExtendedSubmission
 import ebi.ac.uk.model.Submission
 import ebi.ac.uk.persistence.PersistenceContext
 import ebi.ac.uk.util.collections.ifNotEmpty
+import org.springframework.transaction.annotation.Isolation
+import org.springframework.transaction.annotation.Transactional
 
-class SubmissionSubmitter(
+open class SubmissionSubmitter(
```

Comment:
why this class is open now?

Reply:
"The default advice mode for processing `@Transactional` annotations is `proxy`" which requires overriding

https://docs.spring.io/spring/docs/5.2.x/spring-framework-reference/data-access.html#transaction-declarative-annotations

Reply:
yes :) but we use `allopen` kotlin compiler plugin that will do that for us. So adding plugin in subproject will do the trick with not coding changes. (check persistence project for reference )

Reply:
allopen not behaving as documented.